src/manifest.ts

```diff
@@ -174,6 +175,7 @@ interface ReleaserConfigJson {
   'pull-request-header'?: string;
   'pull-request-footer'?: string;
   'separate-pull-requests'?: boolean;
+  'always-update'?: boolean;
```

Comment:
I think this one should be moved to `ManifestConfig` - `ReleaserConfigJson` are the per-component configurations and ` always-update` seems to be a repository-wide configuration.